documentation/jetty-documentation/src/main/asciidoc/operations-guide/modules/module-eex-webapp.adoc

```diff
@@ -0,0 +1,25 @@
+//
+// ========================================================================
+// Copyright (c) 1995-2022 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+[[og-module-eex-webapp]]
```

Comment:
Maybe I like better `eeN`, the difference in case of the `N` is more understandable, wdyt?